migrate_and_seed_database.js

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
 
-const sqlite3 = require('sqlite3').verbose();
-const db = new sqlite3.Database('./db/gddy.sqlite');
+const db = require('better-sqlite3')('./db/gddy.sqlite', {verbose: console.log})
```

Comment:
The sqlite library would not build for me, so I switched to `better-sqlite3` and updated this script to use it.